service/matching/pri_task_writer.go

```diff
@@ -52,12 +53,13 @@ type (
 
 	// priTaskWriter writes tasks persistence split among subqueues
 	priTaskWriter struct {
-		backlogMgr  *priBacklogManagerImpl
-		config      *taskQueueConfig
-		db          *taskQueueDB
-		logger      log.Logger
-		appendCh    chan *writeTaskRequest
-		taskIDBlock taskIDBlock
+		backlogMgr      *priBacklogManagerImpl
+		config          *taskQueueConfig
+		db              *taskQueueDB
+		logger          log.Logger
+		appendCh        chan *writeTaskRequest
+		taskIDBlock     taskIDBlock
+		lastTaskIDBlock taskIDBlock // copy of the last taskIDBlock for safe concurrent access via currentTaskIDBlock()
```

Comment:
last/current naming is confusing, can we use the same name for the data and the accessor except with a "get"?